data/synth_age_range_dim.csv

```diff
@@ -16,4 +16,4 @@
 15,"70 - 74",70,74
 16,"75 - 79",75,79
 17,"80 - 84",80,84
-18,"85 - 110",85,110
+18,"85 - 200",85,200
```

Comment:
Make this "85+" instead of "85 - 200". It looks cleaner and then Mike won't have to modify it when he reads these age ranges to use in the UI.